Apps/CocoaPods-FCM/src/View/CustomEventView.swift

```diff
@@ -27,13 +27,19 @@ struct CustomEventView: View {
                 }.padding([.vertical], 40)
 
                 ColorButton("Send Event") {
-                    showConfirmationAlert = true
+                    var alertMessage = "Event sent successfully!"
+
+                    if eventName.isEmpty {
+                        alertMessage += "\n\n Note: Empty event name might result in expected behavior with the SDK."
+                    }
+
+                    self.alertMessage = alertMessage
                 }.setAppiumId("Send Event Button")
             }.padding([.horizontal], 20)
-                .alert(isPresented: $showConfirmationAlert) {
+                .alert(isPresented: .notNil(alertMessage)) {
                     Alert(
                         title: Text(""),
-                        message: Text("Event sent successfully"),
+                        message: Text(alertMessage!),
```

Comment:
Are we still showing alert messages ? I thought we moved to toast messages 🤔

Reply:
Sorry, I do not recall this decision? 

I do like toasts compared to an Alert. Adding toasts to this app would require some refactoring and prototyping since I am not sure how to do it in SwiftUI yet. 

I could circle back to this after my other higher priority tickets if the team finds it valuable.

Reply:
[Here is the slack thread](https://customerio.slack.com/archives/C05C2HF5VB2/p1686227536321599) that talks about this decision in case you missed it.

Reply:
And [this conversation](https://customerio.slack.com/archives/C05C2HF5VB2/p1686227048608089) too talks about toast.